docs/tech_specs/dup_v2_alert_widget_backend.md

```diff
@@ -0,0 +1,151 @@
+- Feature Name: `dup_v2_alert_widget_backend`
+- Start Date: 2023-01-30
+- RFC PR: [#1616](https://github.com/mbta/screens/pull/1616)
```

Comment:
Another GH markdown feature: If you just type #1616, it will recognize it as a PR/issue reference number for this repo. So this can just be
```suggestion
- RFC PR: #1616
```

Reply:
Hmm. For some reason, that doesn't seem to work in this file. I've seen it work many times in comments, but it doesn't seem to have that in md files themselves.